.ado/jobs/build-test-rntester.yml

```diff
@@ -89,88 +89,77 @@ parameters:
 
 jobs:
   - ${{ each slice in parameters.appleBuildMatrix }}:
-      - job: ${{ slice.name }}
-        displayName: ${{ slice.friendly_name }}
-        pool:
-          vmImage: $(vmImageApple)
-        timeoutInMinutes: 90
-        cancelTimeoutInMinutes: 5
-        steps:
+    - job: ${{ slice.name }}
+      displayName: ${{ slice.friendly_name }}
+      pool:
+        vmImage: $(vmImageApple)
+      timeoutInMinutes: 90
+      cancelTimeoutInMinutes: 5
+      steps:
```

Comment:
Why the indentation change?

Reply:
Also, Prettier should have caught these but that's a mystery for another day.